packages/@sanity/types/src/schema/defineTypes.ts

```diff
@@ -0,0 +1,83 @@
+import {PreviewConfig} from './preview'
+import {InitialValueProperty, Schema, SchemaValidationValue} from './types'
+
+export interface DefineOptions<
+  TStrict extends StrictDefinition,
+  TAlias extends Schema.Type | undefined
+> {
+  /**
+   * `strict: false` allows unknown properties in the schema.
+   * Use this when adding customizations to the schema that are not part of sanity core
```

Comment:
Perhaps add a note here suggesting extending the interface with options?

Reply:
Adding a chapter on declaration type merging with examples.